### PR TITLE
Taking tabbedtab.css into tabulator.scss

### PR DESCRIPTION
**Pull request by megoth**
Should only output one file: mash.css

This should solve https://github.com/solid/solid-panes/issues/149

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8"/>
     <title><%= htmlWebpackPlugin.options.title %></title>
-    <link type="text/css" rel="stylesheet" href="https://solid.github.io/solid-panes/style/tabbedtab.css" />
```
Comment (Vinnl) [inline src/index.html]: :tada:

**Comment by timbl** [comment]
I don't think we should have to refer to tabulator any more.  The file should be  mash.scss or makbe solid-panes.scss but not tabulator. And the title "Some styles for the tabulator" should be "some styles or the solid-ui solid-panes stack" maybe.  As you know I think we shoiuld move solid-ui style at least out of the css files and into JS variables.

**Comment by megoth** [comment]
> I don't think we should have to refer to tabulator any more. The file should be mash.scss or makbe solid-panes.scss but not tabulator. And the title "Some styles for the tabulator" should be "some styles or the solid-ui solid-panes stack" maybe. As you know I think we shoiuld move solid-ui style at least out of the css files and into JS variables.

Could we replace tabulator with "Solid Data Browser" perhaps, as that is the current cover-it-all for mashlib/solid-panes/solid-ui+++ stack?

**Comment by megoth** [comment]
> As you know I think we shoiuld move solid-ui style at least out of the css files and into JS variables.

I've this in the original issue as well.

**Comment by megoth** [comment]
@timbl have renamed files and moved some stuff that were unnecessary (I think). There are still some tabulator specific selectors that I did not want to touch, i.e. `form#TabulateForm` and `#TabulatorStatusWidget`.

Does this look more what you wanted?

**Comment by megoth** [comment]
I'll fix the conflicting files, then I'll merge ^_^

**Pull request merged**